jwql/database/database_interface.py

```diff
@@ -241,8 +241,14 @@ class : obj
     data_dict = {}
     data_dict['__tablename__'] = class_name.lower()
 
+    inst = data_dict['__tablename__'].split('_')[0]
```

Comment:
@tnking97 Generally speaking, it is best to be as explicit as possible with names.  I would suggest changing these variable names to `instrument`, `instrument_anomalies`, `ANOMALIES_PER_INSTRUMENT`, etc. 

It seems silly, but a minor thing like that can really help with the readability of the code.